packages/next/src/server/lib/trace/tracer.ts

```diff
@@ -174,12 +177,14 @@ class NextTracerImpl implements NextTracer {
           }
 
     if (
-      !NextVanillaSpanAllowlist.includes(name) &&
+      !NextVanillaSpanAllowlist.includes(type) &&
       process.env.NEXT_OTEL_VERBOSE !== '1'
     ) {
       return fn()
     }
 
+    const traceName = options.tracerName ?? type
```

Comment:
what is tracer name? a custom span name in this casE?

Reply:
also, traceName vs tracerName, which one is the correct one?

Reply:
Oh, I missed that `r` at the end.

Thanks. I reused the existing prop that wasn't used anywhere, but you probably wanted to use it for `tracerName`. I'll delete that prop and add a new one `spanName`.